src/features/ticket/hooks/useOrderHook.ts

```
@@ -50,13 +50,19 @@ export const useOrderTicket = () => {
 // qr 스캔
 export const useTicketQrCodeValidate = () => {
   return useMutation({
-    mutationFn: ({ orderId, sig }: { orderId: number; sig: string }) =>
-      ticketQrCode(orderId, sig),
+    mutationFn: ({ orderId, sig }: { orderId: number; sig: string }) => ticketQrCode(orderId, sig),
     onSuccess: () => {
       alert('체크인 성공!');
     },
-    onError: () => {
-      alert('체크인 실패했습니다. 다시 시도해주세요.');
+    onError: (error: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)